Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs

```diff
@@ -54,11 +54,11 @@ public NetqRabbitRpcServer(INetqRabbitRpcConnection netqRpcRabbitConnection, ILo
             VerifyServerHasntStartedYet();
             if (_netqRabbitRpcConnection.RpcMessageType == RpcMessageType.Abstract)
             {
-                _respondHandler = _netqRabbitRpcConnection.Bus.RespondAsync<TRequest, TRespond>(callback);
```

Comment:
Why you moved the TRequest, Trespond here but kept it in other places , be consistent about it, i think that explicit is better

Reply:
f